extensions/pkg/controller/healthcheck/config/v1alpha1/types.go

```diff
@@ -23,4 +27,19 @@ type HealthCheckConfig struct {
 	// already running on them).
 	// defaults to 30 sec
 	SyncPeriod metav1.Duration `json:"syncPeriod"`
+	// ShootRESTOptions allow overwriting certain default settings of the shoot rest.Client
```

Comment:
Do you mean `rest.RESTClient` or `rest.Config`? I am not sure that `rest.Client` exists.

```suggestion
	// ShootRESTOptions allow overwriting certain default settings of the rest.Config used for the Shoot client.
```

Reply:
`rest.Config` it is